### PR TITLE
sriov pkg: add DisablePlugins and GetTotalVFs methods

### PR DESCRIPTION
**Pull request by evgenLevin**
Added  :
1. WithDisablePlugins  and RemoveDisablePlugins funcs for the SriovOperatorConfig CR
2. GetTotalVFs func for the SriovNetworkNodeState CR

### CODE REVIEW

**Review by kononovn — DISMISSED** [review]
/lgtm

File: pkg/sriov/networknodestate_test.go [hunk 1]
```diff
@@ -328,6 +328,42 @@ func TestNetworkNodeStateGetNumVFs(t *testing.T) {
 	}
 }
 
+func TestNetworkNodeStateGetTotalVFs(t *testing.T) {
+	testCases := []struct {
+		netInterface srIovV1.InterfaceExts
+		upInterface  srIovV1.InterfaceExts
```
Comment (achuzhoy) [inline pkg/sriov/networknodestate_test.go:334]: I don't see it being used anywhere. Needed?

File: pkg/sriov/operatorconfig.go [hunk 2]
```diff
@@ -237,6 +243,47 @@ func (builder *OperatorConfigBuilder) WithConfigDaemonNodeSelector(
 	return builder
 }
 
+// WithDisablePlugins configures disablePlugins in the SriovOperatorConfig.
+func (builder *OperatorConfigBuilder) WithDisablePlugins(plugins []string) *OperatorConfigBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Configuring disablePlugins %v to SriovOperatorConfig object %s",
```
Comment (achuzhoy) [inline pkg/sriov/operatorconfig.go:252]:
```suggestion
	glog.V(100).Infof("Configuring disablePlugins %v in SriovOperatorConfig object %s",
```

File: pkg/sriov/networknodestate_test.go (hunk 1 above)
Comment (evgenLevin) [reply pkg/sriov/networknodestate_test.go]: removed

File: pkg/sriov/operatorconfig.go (hunk 2 above)
Comment (evgenLevin) [reply pkg/sriov/operatorconfig.go]: changed

**Review by achuzhoy — APPROVED** [review]

**Pull request merged**